File: Fácil/Onboarding/Onboarding.cpp
#include <iostream>
using namespace std;

int main() {
  while (1) {
    string enemy_1; // name of enemy 1
    cin >> enemy_1; cin.ignore();
    int dist_1; // distance to enemy 1
    cin >> dist_1; cin.ignore();
    string enemy_2; // name of enemy 2
    cin >> enemy_2; cin.ignore();
    int dist_2; // distance to enemy 2
    cin >> dist_2; cin.ignore();
    cout << ((dist_1 < dist_2) ? enemy_1 : enemy_2) << endl;
  }
}

File: Easy/Onboarding/Onboarding.swift
import Glibc
import Foundation

public struct StderrOutputStream: TextOutputStream {
    public mutating func write(_ string: String) { fputs(string, stderr) }
}
public var errStream = StderrOutputStream()

while true {
    let enemy1 = readLine()!
    let dist1 = Int(readLine()!)!
    let enemy2 = readLine()!
    let dist2 = Int(readLine()!)!

    if (dist1 < dist2)
    {
        print(enemy1)
    }
    else
    {
        print(enemy2)      
    }
}
